extension/webview/TransactionView.ts

```diff
@@ -38,8 +40,8 @@ export class ReactView extends View {
     }
 
     loadComponent(panel: vscode.WebviewPanel): void {
-        // Tell the React app to open up the component where the route is '/one'
         panel.webview.postMessage('/transaction');
+        panel.webview.postMessage(this.appState);
```

Comment:
Could do something like:
`panel.webview.postMessage({path: '/transaction', state: this.appState});`

Reply:
Might be worth removing the stuff on lines 33-37 as it's commented out.